client/js/views/GameView.vue

```diff
@@ -1253,30 +1249,69 @@ export default {
 .history-container {
   grid-area: history;
 }
+
 #field-right {
   height: 100%;
+  padding-top: 0.5rem;
 
   #history,
   #card-preview {
     margin: 0 auto;
-    padding: 5px;
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    border: 1px solid #fff;
+    padding: 0.5rem;
     width: 80%;
     max-width: 80%;
     height: 100%;
+    border-radius: 20px;
+    box-shadow: 0px 0px 10px #111111;
```

Comment:
It would probably be more consistent to give this an elevation class rather than a custom box shadow

Reply:
Check the Vuetify docs: https://vuetifyjs.com/en/styles/elevation/

Elevation is a material design concept of using specific box shadows to suggest depth and clarify what's on top of what. Since were generally using material design it'd make sense to use an elevation shadow that'd be more consistent with the rest of the app. Vuetify provides styles that we can apply using classes like `class="elevation-10"` to make this easy

Reply:
I agree with this callout. ☝🏼

Reply:
I added an elevation-10 and removed border-shadow from the element.